examples/src/components/MonthPickerExample.jsx

```diff
@@ -0,0 +1,178 @@
+// (C) 2007-2019 GoodData Corporation
+/* eslint-disable react/jsx-closing-tag-location */
+import React, { Component } from "react";
+import { AfmComponents, ErrorComponent } from "@gooddata/react-components";
+import DatePicker from "react-datepicker";
+import moment from "moment";
+
+import "@gooddata/react-components/styles/css/main.css";
+import "react-datepicker/dist/react-datepicker.css";
+
+import {
+    totalSalesIdentifier,
+    monthOfYearDateIdentifier,
+    dateDatasetIdentifier,
+    projectId,
+} from "../utils/fixtures";
+
+const { ColumnChart } = AfmComponents;
+
+const dateFormat = "YYYY-MM-DD";
+
+export class MonthPickerExample extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            from: moment("2016-01-01", dateFormat),
+            to: moment("2017-01-01", dateFormat),
+            error: null,
+        };
+
+        this.onFromChange = this.onFromChange.bind(this);
+        this.onToChange = this.onToChange.bind(this);
+        this.onDateChange = this.onDateChange.bind(this);
+    }
+
+    onFromChange(value) {
+        this.onDateChange("from", value);
+    }
+
+    onToChange(value) {
+        this.onDateChange("to", value);
+    }
+
+    onDateChange(prop, value) {
+        const { from, to } = this.state;
+        const newState = {
+            from,
+            to,
+            error: null,
+        };
+        newState[prop] = value;
+
+        if (newState.to.isSameOrAfter(newState.from)) {
+            this.setState(newState);
+        } else {
+            this.setState({
+                error: '"From" date must come before "To" date.',
+            });
+        }
+    }
+
+    onLoadingChanged(...params) {
+        // eslint-disable-next-line no-console
+        console.info("MonthPickerExample onLoadingChanged", ...params);
+    }
+
+    onError(...params) {
+        // eslint-disable-next-line no-console
+        console.info("MonthPickerExample onLoadingChanged", ...params);
+    }
+
+    monthDiff(date) {
```

Comment:
You are using `moment` library in this example file. Imho you could write content of the function in one or two lines if you use it here as well ;)
http://momentjs.com/docs/#/displaying/difference/